sync/src/chain.rs

```diff
@@ -80,9 +82,7 @@ const NODE_DATA_PACKET: u8 = 0x0e;
 const GET_RECEIPTS_PACKET: u8 = 0x0f;
 const RECEIPTS_PACKET: u8 = 0x10;
 
-const NETWORK_ID: U256 = ONE_U256; //TODO: get this from parent
-
-const CONNECTION_TIMEOUT_SEC: f64 = 10f64;
+const CONNECTION_TIMEOUT_SEC: f64 = 5f64;
```

Comment:
5 Seconds is enough for the current network. Most clients respond within 1 second. Lower timeout prevent temporary download stalling.